CMakeLists.txt

```
@@ -21,6 +21,7 @@ if (POLICY CMP0075)
 endif()
 
 project(DALI CXX)
+set(DALI_TOP_DIR ${PROJECT_SOURCE_DIR})
```

Following usual convention, that would be
```suggestion
set(DALI_ROOT ${PROJECT_SOURCE_DIR})
```